web-app/js/smartR/_angular/viz/d3Heatmap.js

```diff
@@ -190,15 +190,8 @@ window.smartRApp.directive('heatmapPlot', ['smartRUtils', 'rServeService', funct
                     d3.selectAll('.patientID').classed('highlight', false);
                     d3.selectAll('.uid').classed('highlight', false);
                     tooltip.style('visibility', 'hidden');
-                })
-                .on('click', function (d) {
-                    var genes = d.UID.split("--");
-                    genes.each(function (gene) {
-                        var url = 'http://www.genecards.org/cgi-bin/carddisp.pl?gene=' + gene;
-                        window.open(url);
-                    });
                 });
```

Comment:
most likely, but I'm not a lawyer so I do not want to write a text for commercial entities. I will ask eTRIKS